partcad/src/partcad/sentry.py

```
@@ -0,0 +1,135 @@
+import inspect
+import logging
+from opentelemetry import trace
+from contextlib import asynccontextmanager
+import sentry_sdk
+from opentelemetry import trace
```

Ruff (0.8.2)

Redefinition of unused `trace` from line 3

Remove definition: `trace`

(F811)